index.js

```diff
@@ -492,6 +498,25 @@ function getStyleFunction(glStyle, source, resolutions, onChange) {
           }
         }
 
+        if (type == 'Point' && 'circle-radius' in paint) {
+          style = new ol.style.Style({
```

Comment:
This will create a new `ol.style.Style` instance for every rendered feature, which is currently very inefficient in OpenLayers. Better to cache the style.